### ISSUE TITLE
Suggestion: Questers finishing on other NPCs

### ISSUE THREAD

**Issue by Fureniku** [CitizensDev/Citizens#499]
Firstly, sorry if this is already possible. I posted on the forums but didn't get a reply :(

Anyway, can we start a quest with one NPC, and finish on another? So a quest would be to "go talk to X" and when you talk to him, he rewards you and the quest is finished.

**Comment by Loserman778** [comment]
Yes i believe it is possible. I'm no pro with quests, but i think if you can do one of two things, add a 2nd main objective, or u can make a second reward which unlocks a quest

**Comment by Fureniku** [comment]
I think you might have gotten confused. Right now all I want is a simple 1-step quest which starts with one NPC and finishes with another, only one objective and one quest.

For example, you talk to someone in a pub. The suggest "You should go get a drink!" (Quest started) You go to the bar tender and get a drink (Quest finished) - going back to tell him you got the drink wouldn't work, because you wouldn't have the drink until you spoke to him again, so it has to finish at the bar.

**Comment by Loserman778** [comment]
Hmm, i see what your saying, i can't really help you though. I'm not sure.

**Comment by byklop** [comment]
As far as I know this is not possible at this moment. But probably it could be implemented to work as another type of reward (when rewards node is used inside an objective node).

Another option is if there is no reward node at the end of quest (parallel to objective node) quest would finish without interaction with questgiver.

Feature like this would definitely give us much more possibilities for quests than we have atm.
